nodejs/eks/nodegroup.ts

```diff
@@ -131,6 +131,11 @@ export interface NodeGroupBaseOptions {
      * public IPs.
      */
     nodeAssociatePublicIpAddress?: boolean;
+
+    /**
+     * Desired Kubernetes master / control plane version. If you do not specify a value, the latest available version is used.
+     */
+    version?: pulumi.Input<string>;
```

Comment:
I’d suggest for now we don’t add the `version` part of this change - and open an issue to follow up with that. 

The primary thing this version should be used for is the master/control plane as the comment notes - but it is not being used for that yet. 

Also - the way platform versions are specified for EKS is different than how this is being used, and we’ll want to be careful to expose the EKS version concept faithfully. 

This is also not strictly necessary to fix the current high-pri break, and adds some risk to the fix for that.

Reply:
The version is being passed into the `aws.eks` cluster [constructor](https://github.com/pulumi/pulumi-aws/blob/master/sdk/nodejs/eks/cluster.ts#L69) in https://github.com/pulumi/pulumi-eks/pull/85/files#diff-0dfaec1e9f3a8568a8e2516febe83a69R521.

This is also then leveraged to assure the worker node AMI matches the master version set, in the `getAmi` filter changes below, if set. 

Could you elaborate why you think it's not being used to set the master / control plane version?

Reply:
Ah - sorry - I misread the code.  Looking again and reviewing the upstream provider `version` specification - this looks exactly right.